.drone.star

```diff
@@ -235,6 +235,10 @@ def build_client(image = OC_CI_CLIENT, ctest = True):
     cmake_options = '-G"%s" -DCMAKE_C_COMPILER="%s" -DCMAKE_CXX_COMPILER="%s" -DCMAKE_BUILD_TYPE="%s"'
```

Comment:
Can you use /usr/clang /usr/clang++ (else it uses the clang provided by craft)

Reply:
Now I only used `cmake -G"Ninja" -DBUILD_TESTING=OFF -S ..` while building client using craft installed libraries